content/docs/localization/joining-the-localization-team.md

```diff
@@ -7,40 +7,111 @@ aliases:
 ---
 
 {{< admonition info >}}
+If you found a **translation mistake/error** in Scratch Addons, please [send feedback](/feedback) about it.

```

Comment:
Could creating an issue be an alternative here?

Reply:
Yep, that's also okay. But I don't think it makes sense to make this paragraph unnecessarily long. The feedback form is often enough to explain which string is wrongly translated. We already received a few feedback messages like those, even without explicitly suggesting users to report translation mistakes through feedback.